scripts/events_module/relationship/romantic_events.py

```diff
@@ -467,24 +468,50 @@ def handle_breakup(cat_from: Cat, cat_to: Cat) -> bool:
             relationship_to = cat_to.create_one_relationship(cat_from)
 
         # These are large decreases - they are to prevent becoming mates again on the same moon.
-        relationship_to.romantic_love -= 15
-        relationship_from.romantic_love -= 15
-        relationship_to.comfortable -= 10
-        relationship_from.comfortable -= 10
-        if had_fight:
-            relationship_to.romantic_love -= 5
-            relationship_from.romantic_love -= 5
+        if breakup_type == "had_fight":
+            relationship_to.romantic_love -= 15
+            relationship_from.romantic_love -= 15
             relationship_from.platonic_like -= 10
             relationship_to.platonic_like -= 10
             relationship_from.trust -= 10
             relationship_to.trust -= 10
             relationship_to.dislike += 10
             relationship_from.dislike += 10
-
-        if had_fight:
-            text = i18n.t("hardcoded.breakup_angy")
-        else:
-            text = i18n.t("hardcoded.breakup_chill")
+        elif breakup_type == "decided_to_be_friends":
+            relationship_to.romantic_love -= 30
+            relationship_from.romantic_love -= 30
+            relationship_from.platonic_like += 30
+            relationship_to.platonic_like += 30
+            relationship_from.trust += 20
+            relationship_to.trust += 20
+            relationship_to.comfortable += 5
+            relationship_from.comfortable += 5
+        elif breakup_type == "lost_feelings":
+            relationship_to.romantic_love -= 30
+            relationship_from.romantic_love -= 30
+            relationship_from.platonic_like -= 10
+            relationship_to.platonic_like -= 10
+            relationship_to.comfortable -= 10
+            relationship_from.comfortable -= 10
+        elif breakup_type == "bad_breakup":
+            relationship_to.romantic_love -= 20
+            relationship_from.romantic_love -= 15
+            relationship_from.platonic_like -= 10
+            relationship_to.platonic_like -= 15
+            relationship_from.trust -= 20
+            relationship_to.trust -= 25
+            relationship_to.comfortable -= 20
+            relationship_from.comfortable -= 20
+            relationship_to.dislike += 10
+            relationship_from.dislike += 5
+        elif breakup_type == "chill_breakup":
+            relationship_to.romantic_love -= 15
+            relationship_from.romantic_love -= 15
+            relationship_to.comfortable -= 10
+            relationship_from.comfortable -= 10
+
+        text = choice(RomanticEvents.BREAKUP_STRINGS[breakup_type])
+            text = event_text_adjust(Cat, text, main_cat=cat_from, random_cat=cat_to)
```

Comment:
```suggestion
        text = choice(RomanticEvents.BREAKUP_STRINGS[breakup_type])
        text = event_text_adjust(Cat, text, main_cat=cat_from, random_cat=cat_to)
```
just fixing the indentation here to avoid an error, should be the last thing though!